week1/3-website/css/style.css

```diff
@@ -6,3 +6,176 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+
+body {
+    margin: 20px;
+    line-height: 1.5; /* to create gaps between lines for easy readying*/
+}
+
+ header, footer {
+     color: rgb(17, 80, 17);
+ }
+ 
+ /*BUG: I tried header:not(ul) to select everything in header but not ul but it didn't work*/
```

Comment:
I think `header:not(ul)` is the same as just `header`. You are basically saying "select all `header` elements that ar not `ul`". Here you would want to use something like `header > *:not(ul)` but it's confusing and something that's probably you want to avoid.